packages/core/core/src/RequestTracker.js

```diff
@@ -345,6 +351,9 @@ export class RequestGraph extends ContentGraph<
     for (let parentNode of parentNodes) {
       this.invalidateNode(parentNode, reason);
     }
+
+    // If the node is invalidated, the cached request chunk on disk needs to be re-written
+    this.cachedRequestChunks.delete(Math.floor(nodeId / NODES_PER_BLOB));
```

Comment:
I think we may need to do this from `Request.completeRequest` as well? I think it's possible that a node is invalidated but its request hasn't been re-run yet. Once it is re-run we'd want to update it's result in the cache.